tests/typings/copy.js

```diff
@@ -33,6 +33,11 @@ const packages = {
     '@types/jasmine/ts3.1': 'packages/wdio-jasmine-framework/node_modules/@types/jasmine/ts3.1',
     '@wdio/jasmine-framework': 'packages/wdio-jasmine-framework',
 
+    '@types/request': 'packages/webdriver/node_modules/@types/request',
+    '@types/caseless': 'packages/webdriver/node_modules/@types/caseless',
+    '@types/tough-cookie': 'packages/webdriver/node_modules/@types/tough-cookie',
+    'form-data': 'packages/webdriver/node_modules/@types/request/node_modules/form-data',
```

Comment:
Can we add a typings test here:

- https://github.com/webdriverio/webdriverio/blob/v5/tests/typings/webdriverio/async.ts#L139
- https://github.com/webdriverio/webdriverio/blob/v5/tests/typings/sync/config.ts

Reply:
It works for the sync test but when I'm attaching the config to `async` it doesn't recognize `transformRequest`. Also it does not recognize any of the other config options coming from `webdriver` like `headers` or `agent`. Any hint what has to be changed there?

Reply:
I am currently looking into this. I don't want to make this a blocker for this PR.